README.md

```diff
@@ -98,142 +113,148 @@ oc login -u developer -p developer https://localhost:8443
-* sparkdriver - recv\_temperature, send\_max
-* thermostat - recv\_max, send\_control*
+![MyApp](images/myapp1.png)
 
-You can edit the groups for each user by editing the user and clicking on the groups tab. This ensures that none of the components can access addresses they should not access.
+Once the provisioning is complete, we can login to the console and create the addresses we need for
+the workshop.
 
-#### Creating messaging addresses
+### Creating messaging addresses
 
 In EnMasse, you have the concepts of address spaces and addresses.
 
 An address space is a group of addresses that can be accessed through a single connection (per
 protocol). This means that clients connected to the endpoints of an address space can send messages
 to or receive messages from any address it is authorized to send messages to or receive messages
 from within that address space. An address space can support multiple protocols, which is defined by
-the address space type. In this workshop, we only have 1 address space 'default' of type 'standard'.
+the address space type.
+
+Each messaging service provisioned in the service catalog creates a new address space. Conceptually,
+an address space may share messaging infrastructure with other address spaces.
 
 An address is part of an address space and represents a destination used for sending and receiving
 messages. An address has a type, which defines the semantics of sending messages to and receiving
-messages from that address.
+messages from that address. An address also has a plan, which determines the amount of resources
+provisioned to support the address.
 
 In the 'standard' address space, we have 4 types of addresses.
 
-   * **multicast** : 'direct' one-to-many using dispatch router
-   * **anycast** : 'direct' peer-2-peer using dispatch router
-   * **queue** : queue on broker
-   * **topic** : pub/sub on broker
+   * **multicast** : 'direct' one-to-many
+   * **anycast** : 'direct' peer-2-peer
+   * **queue** : queue
+   * **topic** : pub/sub
+
+### Creating addresses for this workshop
 
-##### Creating addresses for this workshop
+Login to the messaging console URL given by the provisioned service. You should be redirected to the authentication service. 
 
-Go to the console, and locate the 'console' route. Click on the link to get to the EnMasse console.
+![Authentication service](images/auth_service.png)
 
-Create an addresses for your IoT sensors to report metrics on:
+On the side of the login form, you can see a button named "OpenShift". Click on that to authenticate your user using your OpenShift credentials.
+
+Once logged in, create the following addresses:
 
    * _temperature_ : type topic - used by devices to report temperature
    * _max_ : type anycast - used by Spark driver to report the max temperature
    * _control/deviceX_ : type topic - used to send control messages to devices. Per-device control messages will be sent to control/$device-id
```

Comment:
@lulf  are we going to use sharded or pooled topic ? Is there something not working with MQTT with pooled topic IIRC ?

Reply:
@lulf I don't remember if the hierarchical topics work fine, so it's enough to create "control" address or we need "control/device1", "control/device2" and so on.

Reply:
@ppatierno MQTT only works with sharded topic at present. Regarding hierarchical topics I think it is not working but you can test. I have been creating control/device1 etc. when testing because I assumed it was not working.